packages/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -17,11 +19,15 @@ exports.inputs = {
     addressCity: () => $('[data-test-id="input-address-city"]'),
     addressPostcode: () => $('[data-test-id="input-address-postcode"]')
 };
+exports.orderTimeDropdownOptions = () => $$('[data-test-id="form-select"] select option');
```

Comment:
Can we rename this `getOrderTimeDropdownOptions`?

Reply:
Having said that, it might be worth rethinking this. What're you thoughts @Festivejelly?

We shouldn't be making the elements accessible from the spec.

Reply:
yep sure thing! :)

Reply:
Can we also add a JSDoc description to this function if we're going to keep it?